code/modules/clothing/under/costume.dm

```diff
@@ -290,3 +290,75 @@
 	alt_covers_chest = TRUE
 	sensor_mode = SENSOR_COORDS
 	random_sensor = FALSE
+
+/obj/item/clothing/under/costume/metalgear
+	name = "\improper sneaking suit"
+	desc = "It's made of a strong material developed by the Soviet Union centuries ago which provides robust protection."
+	icon_state = "solid"
+	item_state = "solid"
+	body_parts_covered = CHEST|GROIN|ARMS|LEGS
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 30, "acid" = 60)
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/biker
+	name = "biker's outfit"
+	desc = "You're not a nice person, are you?"
+	icon_state = "biker"
+	item_state = "biker"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/jacket
+	name = "richard's outfit"
+	desc = "Do you know what time it is?"
+	icon_state = "jacket"
+	item_state = "jacket"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/greaser
+	name = "greaser outfit"
+	desc = "The one that you want!"
+	icon_state = "greaser"
+	item_state = "greaser"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/greaser/snake
+	name = "greaser outfit"
+	desc = "Tunnel Snakes Rule!"
+	icon_state = "greaser_snake"
+	item_state = "greaser"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/greaser/spider
+	name = "greaser outfit"
+	desc = "The one that you want!"
+	icon_state = "greaser_spider"
+	item_state = "greaser"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/greaser/cult
+	name = "greaser outfit"
+	desc = "The one that you want!"
+	icon_state = "greaser_cult"
```

Comment:
`[02:27:55] Runtime in unsorted.dm,1216: /obj/item/clothing/under/costume/greaser/cult does not have a valid icon state, icon=icons/obj/clothing/uniforms.dmi, icon_state="greaser_cult"([0x600d469])`

Reply:
o co tu chodzi? greaser uzywają jako item tego samego, przy testach działało bez problemów i w unsorted.dm na linijce 1216 jest takie cos:
![brugh](https://user-images.githubusercontent.com/60329232/100385830-15585b00-3024-11eb-9bcc-524b05a0b9a4.png)

Reply:
```  
  vending (/datum/asset/spritesheet/vending): stack trace("/obj/item/clothing/under/costu...")
  vending (/datum/asset/spritesheet/vending): register()
  vending (/datum/asset/spritesheet/vending): New()
  get asset datum(/datum/asset/spritesheet/vendi... (/datum/asset/spritesheet/vending))
  Assets (/datum/controller/subsystem/assets): Initialize(660471)
```
tu masz cały call stack, no ale wygląda jakby z icon states było coś nie tak, btw bazaltowany font, ale kolory WutFace

Reply:
naprawione, dodałem do uniforms.dmi takie same ikony tylko z inną nazwa dla każdego przedmiotu i juz nie ma runtimes 😃